sdks/Cordova.md

```diff
@@ -0,0 +1,136 @@
+# Apache Cordova AeroGear Services
+
+## Motivation
+
+Provide set of the SDK's that can be used for Cordova based development.
+Provide best performance and user experience on the platform.
+
+## Proposed solution 1
+
+Pure javascript solution for the mobile configuration and orchestration 
+that will allow cooperability between JavaScript and Native.
+Configuration parser and other pure JS code willnot require any native implementation and can be reused for cordova/react native implementations. Individual SDK will may by default be implemented in pure JavaScript or as Cordova/React plugins. 
+
+## Advantages of the solution 1
+
+- Significantly reduced effort to develop and maintain Cordova Support by reusing available components.
+- Aligned with the way cordova works (single plugin to do specific *native* job, integrated with others in JavaScript)
+- Ability to reuse existing cordova plugins that perform native tasks (push notifications, sync, auth)
+- Ability to wrap existing JavaScript solutions and use them without 
+- Ability to select and use only elements that are needed in the app.
+
+See investigation details bellow for more details
+
+## Proposed solution 2
+
+Pure JavaScript implementations (JavaScript SDK) that will we written for the web and abstract from browser support.
```

Comment:
@wtrocki I'm not getting the difference between solution 1 & 2.

1 says `Pure javascript solution for the mobile configuration and orchestration ..... Individual SDK will may by default be implemented in pure JavaScript or as Cordova/React plugins`

2 says `Pure JavaScript implementations ..... each platform will provide wrappers for specific apis if needed`

They sound the same to me.

Reply:
Going to remove proposion nr1 as it's kinda confusing. Proposition nr1 has native code wrappers as plugins. Proposition nr2 is js implementation, but it still allows to use plugins when needed so this is where confusion may come from.

See: https://github.com/aerogear/proposals/pull/25#issuecomment-366304423